cumulusci/tasks/metadata_etl/help_text.py

```diff
@@ -0,0 +1,80 @@
+from cumulusci.core.utils import process_list_arg
+
+
+class SetFieldHelpText(MetadataSingleEntityTransformTask):
+    entity = "CustomObject"
+    task_options = {
+        "fields": {
+            "description": "List of object fields to affect, in Object__c.Field__c form.",
+            "required": True,
+        },
+        **MetadataSingleEntityTransformTask.task_options,
+    }
+
+    def _init_options(self, kwargs):
+        self.task_config.options["api_names"] = "dummy"
+        super()._init_options(kwargs)
+
+        try:
+            float(self.api_version)
+        except ValueError:
+            raise TaskOptionsError(f"Invalid API version {self.api_version}")
+
+        if "fields" not in self.options:
+            raise TaskOptionsError(
+                "The 'fields' option is required, please pass a dictionary with the api_name and help_text keys."
+            )
+        if type(self.options["fields"]) != list or len(self.options["fields"]) == 0:
+            raise TaskOptionsError(
+                "Please populate the fields field with a list of dictionaries containing at minimum one entry with an 'api_name' and 'help_text' keys"
+            )
+        if not all(["api_name" in entry for entry in self.options["fields"]]):
+            raise TaskOptionsError(
+                "The 'api_name' key is required on all entry values."
+            )
+        if not all(["help_text" in entry for entry in self.options["fields"]]):
+            raise TaskOptionsError(
+                "The 'help_text' key is required on all entry values to declare what help text value to insert."
+            )
+        self.api_name_list = defaultdict(list)
+        for entry in process_list_arg(self.options["fields"]):
+            try:
+                obj, field = entry["api_name"].split(".")
+
+                self.api_name_list[self._inject_namespace(obj)].append(
+                    (self._inject_namespace(field), entry["help_text"])
+                )
+            except ValueError:
+                raise TaskOptionsError(
+                    f"api_name {entry} is not a valid Object.Field reference"
+                )
+
+        self.api_names = set(self.api_name_list.keys())
+
+    def _transform_entity(self, metadata: MetadataElement, api_name: str):
+        for entry in self.api_name_list[api_name]:
+            self._modify_help_text(metadata, api_name, entry[0], entry[1])
+        return metadata
+
+    def _modify_help_text(
+        self,
+        metadata: MetadataElement,
+        api_name: str,
+        custom_field: str,
+        help_text: str,
+    ):
+        # Locate the <fields> entry for this field entry.
+        field = metadata.find("fields", fullName=custom_field)
+        if not field:
+            raise TaskOptionsError(
+                f"The field {api_name}.{custom_field} was not found."
+            )
+        try:
+            field.inlineHelpText.text = help_text
```

Comment:
I'm concerned that this could overwrite help text customizations that the admin has made in the org.

Reply:
This is a good point. I was envisioning the use case of this task as being in an explicit opt-in installer, used by admins to update their orgs to the latest shipped help text - i.e., that possibility would be disclosed to them and they'd accept before using it.

I wonder if there's a safer way to do that, though, that still provides utility. One option could be to have the task accept the last-shipped Help Text and check if it is still set before updating it. But there may be situations in NPSP where the help text has changed multiple times, and the end-user org may not have the text we expect even if it has not been modified by the end-user admin.

Reply:
Yeah, I'd be less concerned with it in that context. Maybe the task should have an option for whether it will overwrite existing help text or not?

Reply:
I like that idea. @jofsky How about it - overwrite existing text only if an option is set to True, and default the option to False?

Reply:
That will force authors of flows and installers to be explicit about their intentions.